File: Group6System-main/Group6System-main/ApplicationTier/src/main/java/com/example/applicationtier/models/SavedAccounts.java
package com.example.applicationtier.models;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;
@AllArgsConstructor
@NoArgsConstructor
@Data
@Component
public class SavedAccounts {
    @JsonProperty("saveaccount")
    private Account saveAccount;
    @JsonProperty("accountname")
    private String accountName;
    @JsonProperty("amount")
    private double amount;
}

File: Group6System-main/Group6System-main/BlazorClient/BlazorClient/Models/SavedAccounts.cs
using System;
using System.Text.Json.Serialization;

namespace BlazorClient.Models
{
    public class SavedAccounts
    {
        [JsonPropertyName("account")]
        public Account SavedAccount { get; set; }
        
        [JsonPropertyName("accountname")]
        public String AccountName { get; set; }
        
        [JsonPropertyName("amount")]
        public double Amount { get; set; }

        public SavedAccounts(Account savedAccount, string accountName, double amount)
        {
            SavedAccount = savedAccount;
            AccountName = accountName;
            Amount = amount;
        }
    }
}